### PR TITLE
Cleanup npm dependencies

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,10 +29,7 @@
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-qunit": "^4.0.0",
     "ember-cli-shims": "^1.1.0",
```
Comment (locks) [inline package.json:31]: can you remind me when we need ember-cli-shims?

Comment (Turbo87) [reply package.json:31]: All subdependencies need to have `ember-cli-babel@^6.6.0`. I just checked and we could throw it out, but at the moment Ember CLI will issue an unnecessary warning when you do it so I'd avoid that for now.

Comment (rwjblue) [reply package.json:31]: > at the moment Ember CLI will issue an unnecessary warning when you do it so I'd avoid that for now

FWIW, this is fixed in latest beta branch (so as of ember-cli@2.17 removing ember-cli-shims will not issue a warning if all addons have at least ember-cli-babel@6.6.2).

**Review by rwjblue — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Turbo87) [reply package.json:31]: oh awesome, must have missed that :)

**Pull request merged**